client/src/components/navigation/components/Announcement.tsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from "react";
+import Box from "@material-ui/core/Box";
+import MenuItem from "@material-ui/core/MenuItem";
+import Menu from "@material-ui/core/Menu";
+import { Link as RouteLink } from "react-router-dom";
+
+import { Button } from "../../button/Button";
+
+export const AnnouncementNav = () => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <>
+      <Button
+        color="primary"
+        aria-controls="simple-menu"
```

Comment:
I know I put this in other 'sub menu comp' from initial setup but don't just copy and paste. You can have better naming than `simple-menu`. + update other comp